client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+        "change (i.e. dx/dt = f(W), where W is a vector of symbols which may include x).",
+    
+    variable: "Target variable to be modified by the Rule's formula.",
+    
+    expression: "A Python evaluable mathematical expression representing the right hand side of a rule function.<br>  For Assignment Rules, this represents the right hand "+
+        "side of a scalar equation.<br>  For Rate Rules, this represents the right hand side of a rate-of-change equation.",
+    
+    annotation: "An optional note about a rule."
+  },
+  sbmlComponentsEditor: {
+    annotation: "An optional note about the SBML Component."
+  },
+  modelSettings: {
+    previewSettings: "Preview Settings are applied to the model and are used for the model preview and all workflows.",
+    
+    previewTime: "End time of simulation.",
+    
+    timeUnits: "Save point increment for recording data.",
+    
+    volume: "The volume of the system matters when converting to from population to concentration form. This will also set a parameter 'vol' for use in custom (i.e. "+
+        "non-mass-action) propensity functions."
+  },
+  simulationSettings: {
+    rtol: "Relative tolerance for ode solutions, controls a relative accuracy (number of correct digits).  The solver keeps the local error estimates less than atol + "+
+        "rtol * abs(y).  Value must be greater than 0.0.",
+
+    atol: "Absolute tolerance for ode solutions, if a component of y is approximately below atol, the error only needs to fall within the same atol threshold, and the "+
+        "number of correct digits is not guaranteed.  The solver keeps the local error estimates less than atol + rtol * abs(y).  Value must be greater than 0.0.",
+
+    ttol: "A relative error tolerance value governing tau-leaping tau selections.  Based on Cao, Y.; Gillespie, D. T.; Petzold, L. R. (2006). 'Efficient step size "+
+        "selection for the tau-leaping simulation method' (PDF). The Journal of Chemical Physics. 124 (4): 044109. Bibcode:2006JChPh.124d4109C. doi:10.1063/1.2159468. "+
+        "PMID 16460151  Value must be between 0.0 and 1.0.",
+
+    seed: "The seed for the simulation.  Set to -1 for a random seed.",
+
+    realizations: "The number of times to sample the chemical master equation. Each trajectory will be returned at the end of the simulation.",
+
+    algorithm:"The solver by which to simulate the model.",
+
+    ode: "This algorithm produces a deterministic continuous solution via ODE.",
+
+    ssa: "",
+
+    tauLeaping: "This algorithm calculates multiple reactions in a single step over a given tau step size.  The change in propensities over this step are bounded "+
+        "by bounding the relative change in state, yielding greatly improved run-time performance with very little trade-off in accuracy",
+
+    hybrid: "This algorithm uses a root-finding interpretation of the direct SSA method, along with ODE solvers to simulate ODE and Stochastic systems interchangeably "+
```

Comment:
This hybrid deterministic-stochastic will produce a solution that is 'discrete-stochastic' for small population sizes and 'continuous-deterministic' for large population sizes.